### ISSUE TITLE
ions in GridMAT-MD

### ISSUE THREAD

**Issue by gsaha322** [jalemkul/gridmat-md#1]
If I don't have any ions in my pure DPPC bilayer system, what would be the ions in parameter file? Can I just comment out the ions part?

**Comment by wjallen** [comment]
Yes, I think you should be able to comment it out. Or even leave it as it is in the example - defining an ion that is not present in the system won't hurt it, as long as the name of the ion (e.g. 'NA+') does not match the name of any protein atoms. While the code is looking for protein atoms, it ignores atoms that match the ion name.

https://github.com/jalemkul/gridmat-md/blob/master/GridMAT-MD.pl#L341-L346

**Comment by gsaha322** [comment]
Dear sir,
   When I am using a multiframe gro/pdb file in the input parameter for
GridMAT-MD, five data files for each frame are generated. But in gnuplot,
we can plot only one data file at a time. How can I plot a particular data
file (suppose average thickness data file) for all frames to show a
timewise fluctuation for that property?

Thanks & Regards
Gourab Saha

**Comment by wjallen** [comment]
Hi,

It sounds like you want to take multiple static images and generate one time series / movie from those images. This is outside the scope of GridMAT-MD. I recommend looking at other tools to do this, for example ffmpeg.

https://stackoverflow.com/questions/24961127/how-to-create-a-video-from-images-with-ffmpeg
